test/daterangepicker.tests.js

```diff
@@ -1123,7 +1157,7 @@ define([
                     this.options = options;
                 }
                 TestPlugin.pluginName = 'testPlugin';
-                TestPlugin.prototype.attach = sinon.spy();
+                TestPlugin.prototype.attach = sandbox.spy();
```

Comment:
If `attach` exists on the `proto` of `TestPlugin` this should become `sandbox.spy(TestPlugin.prototype, 'attach')` so it's actually restore-able from the `sandbox`.